hw/BUILD

```diff
@@ -7,7 +7,7 @@ load("//rules:fusesoc.bzl", "fusesoc_build")
 
 # This configuration exposes fusesoc's "verilator_options" option to the
 # command line. This is intended to allow CI to specifically build a single
-# -threaded Verilator model since the build environment there is more resource
+# -threaded Verilated model since the build environment there is more resource
```

Comment:
I don't mind this this change but there is another occurrence of "Verilator model" just below (Line 12). I would try to keep everything aligned. Maybe it's better to just skip this change at the moment and do a clean up after M2.

Reply:
I wanted to make it clear what those options control because there's been confusion in the past. I'll fix the others in this file.